src/Entity/ElectedRepresentative/ElectedRepresentative.php

```diff
@@ -118,16 +118,23 @@ class ElectedRepresentative
     /**
      * @var bool|null
      *
-     * @ORM\Column(type="boolean", nullable=true)
+     * @ORM\Column(type="boolean", options={"default": false})
      */
-    private $isSupportingLaREM;
+    private $isSupportingLaREM = false;
 
     /**
      * @var bool|null
      *
-     * @ORM\Column(type="boolean", nullable=true)
+     * @ORM\Column(type="boolean", options={"default": false})
      */
-    private $hasFollowedTraining;
+    private $isInstancesMember = false;
```

Comment:
can we rename it `$instanceMember`, so the getter become `isInstanceMember()` ? (If so, don't forget to update the migration, or the column name here)

Reply:
There is no more `instanceMember`, the structure has been completely changed.